build.gradle

```diff
@@ -212,7 +220,7 @@ task testPython(type: Exec, dependsOn: shadowJar) {
      commandLine 'python', '-m', 'unittest', 'hail.api1.tests', 'hail.api2.tests', 'hail.typecheck.tests', 'hail.genetics.tests', 'hail.expr.tests', 'hail.utils.tests', 'hail.methods.tests'
      environment SPARK_HOME: sparkHome
      environment PYTHONPATH: '' + projectDir + '/python:' + sparkHome + '/python:' + sparkHome + '/python/lib/py4j-' + py4jVersion + '-src.zip'
-     environment SPARK_CLASSPATH: '' + projectDir + '/build/libs/hail-all-spark.jar'
+     environment PYSPARK_SUBMIT_ARGS: '--conf spark.driver.extraClassPath=' + projectDir + '/build/libs/hail-all-spark.jar --conf spark.executor.extraClassPath=' + projectDir + '/build/libs/hail-all-spark.jar pyspark-shell'
```

Comment:
what is the `pyspark-shell` doing here?

Reply:
See here for example: https://issues.apache.org/jira/browse/SPARK-17387

And the comment towards the bottom: "BTW my workaround above is wrong - "pyspark-shell" needs to be the last argument in "PYSPARK_SUBMIT_ARGS", not the first."

Reply:
This is super janky and I don't like it, but you've convinced me it's right.